extensions/wikia/UserActivity/styles/UserActivity.scss

```diff
@@ -0,0 +1,118 @@
+@import "skins/shared/mixins/arrow";
+@import "skins/shared/mixins/clearfix";
+@import "skins/shared/color";
+
+.no-image {
+  background-color: #f0f0f0;
+  background-image: url(/extensions/wikia/VideoHandlers/images/Brackets.png);	/* inline */
+  background-position: 50px 25px;
+  background-repeat: no-repeat;
+  height: 90px;
+  width: 160px;
+}
+
+/* The Grid ---------------------- */
+.row { width: 940px; max-width: 100%; min-width: 768px; margin: 0 auto; }
```

Comment:
This whole thing needs to be reworked by someone with better CSS skills.  This was just lifted from the foundation grid CSS.  This applies to the design as a whole as well since this hack had no designer participating.  As such don't want to spend much time fixing this CSS that we'll likely toss if this gets a real design.